main.yml

```diff
@@ -817,6 +817,8 @@ patch:
         title: Systems
       - id: packages
         title: Packages
+      - id: baselines
```

Comment:
You have to update the navigation JSON. main.yml is no longer used for chrome navigation. You can follow this doc: https://github.com/RedHatInsights/cloud-services-config#chromebundle-navigationjson. The section also contains a link with a more detailed description.

Reply:
I have mae some changes in the PR in the expected file. The navigation title was reworded to  'Patch set' by UX.